data/avro-schema/src/main/resources/avro/net/corda/data/messaging/mediator/MediadorState.avsc

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
There's a typo in this filename, `MediadorState.avsc`

Reply:
That wouldn't be like me at all 👁️